src/cloud/docker/docker-mftf.md

```diff
@@ -57,33 +59,51 @@ MFTF support is enabled by the [Selenium container]({{site.baseurl}}/cloud/docke
       {:.bs-callout-info}
```

Comment:
I do not like this step at all. I think it is very difficult to follow on screen. I already spent a lot of time reformatting this file and sent to you, so you have my suggestion for it already.